rocketpool-cli/minipool/close.go

```diff
@@ -44,6 +44,7 @@ func closeMinipools(c *cli.Context) error {
 		return nil
 	}
 
+	hasOnly16ETHMps := true
```

Comment:
we should probably print a message and exit before this if `len(details.Details) == 0`